index.css

```diff
@@ -0,0 +1,205 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,400;0,500;0,700;1,800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: "Montserrat Alternates", sans-serif;
+  color: #1d1c1c;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  height: 15vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  gap: 10px;
+  padding: 10px 25px;
+  background: #ada7a7;
+  box-shadow: 0 2px 10px rgba(82, 80, 80, 0.384);
+}
+
+header a.logo {
+  background: rgba(133, 127, 167, 0.5);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 10px;
+  padding: 18px;
+  border-radius: 50px 0 50px 0;
+  margin-top: 2%;
+}
+
+header a.logo i {
+  font-size: 35px;
+  color: rgb(219, 101, 16);
+}
+
+header a.logo span {
+  font-size: 15px;
+  font-weight: 800;
+  text-transform: uppercase;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+a:hover {
+  cursor: pointer;
+}
+
+a:active {
+  background: rgba(139, 133, 172, 0.5);
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  font-size: 20px;
+}
+
+input {
+  width: 50%;
+  border: 2px solid black;
+  background-color: white;
+  padding: 5px;
+  font-size: 20px;
+  display: block;
+  margin-bottom: 20px;
+  border-radius: 3px;
+}
+
+p {
+  font-size: 20px;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80vw;
+}
+
+.addBtn {
+  align-self: flex-end;
+  padding: 5px 10px;
+  font-size: 15px;
+  box-shadow: 3px 3px 2px black;
+  margin-right: 25%;
+}
+
+.date {
+  display: flex;
+  margin-top: 100px;
+}
+
```

Comment:
- [ ] KIndly make sure to increase the padding on top of the date so that we can see the time on the screen. Now it's covered by the navigation background.

  - ### YOUR DESIGN
   ![screencapture-127-0-0-1-5500-index-html-2023-03-06-12_18_16](https://user-images.githubusercontent.com/87197412/223082439-17613314-5e98-4cd3-8566-a81abf1fb658.png)

Reply:
addition of padding on [`date `](https://github.com/Kel-nana/Awesome-Books-Modules/blob/6bd1920a0af1939f64cf79ec2adb44f22f206af4/index.css#L106-L110)